packages/indexing/src/local-index-api.ts

```diff
@@ -38,23 +39,35 @@ async function _getIndexModelArgs(
   const opts: IndexModelArgs = {
     model: modelStreamId,
   }
+  let relationFields = new Set<string>()
 
   if (modelStreamId.type == Model.STREAM_TYPE_ID) {
     const modelState = await core.loadStream(modelStreamId, {})
-    const content = modelState.state.next?.content ?? modelState.state.content
+    const content: ModelDefinition = modelState.state.next?.content ?? modelState.state.content
     Model.assertVersionValid(content, 'major')
-    if (content.interface) {
```

Comment:
Can you explain why this is not needed here anymore? `ModelDefinition` can still have the interface property, wouldnt this cause interfaces to be indexed now?

Reply:
Yes, the check is still there, just moved here: https://github.com/ceramicnetwork/js-ceramic/pull/3103/files#diff-2884de7343db780f37821fd5a33c5baf2dda390b70387db7c0e610550e9c769dR53